static/app/components/replays/replayController.tsx

```diff
@@ -3,14 +3,25 @@ import styled from '@emotion/styled';
 
 import Button from 'sentry/components/button';
 import ButtonBar from 'sentry/components/buttonBar';
+import {
+  onlyUserActions,
+  transformCrumbs,
+} from 'sentry/components/events/interfaces/breadcrumbs/utils';
 import CompactSelect from 'sentry/components/forms/compactSelect';
 import {useReplayContext} from 'sentry/components/replays/replayContext';
 import useFullscreen from 'sentry/components/replays/useFullscreen';
-import {IconArrow, IconPause, IconPlay, IconRefresh, IconResize} from 'sentry/icons';
+import {formatTime, relativeTimeInMs} from 'sentry/components/replays/utils';
+import {
+  IconArrow,
+  IconNext,
+  IconPause,
+  IconPlay,
+  IconRefresh,
+  IconResize,
+} from 'sentry/icons';
 import {t} from 'sentry/locale';
 import space from 'sentry/styles/space';
-
-import {formatTime} from './utils';
+import {getNextUserAction} from 'sentry/utils/replays/getCurrentUserAction';
```

Comment:
kinda confusing that we're importing `getNextUserAction` from `getCurrentUserAction`

Reply:
yes. The current function in there is being changed into 'getPreviousAction` as part of https://github.com/getsentry/sentry/issues/34554... which... that method name would've help make the ticket easier to describe.

I avoided moving the file to prevent conflicts. But I can rebase this afterwards and tidy up at that time.